README.md

```diff
@@ -5,17 +5,29 @@ Blaze4D is a Fabric mod that changes Minecraft's rendering engine to use the Vul
 Early Development and is **NOT** intended for use by the faint-hearted. Support for Blaze4D can be found in the #support
 Discord channel.
 
-Note: Sodium, Iris or Canvas probably don't work yet, so don't bother trying. We're trying to add support for them ourselves
+## Things that don't work
+- Sodium, Iris or Canvas probably don't work yet, so don't bother trying. We're trying to add support for them ourselves
+- Fabulous graphics
+- MacOS
+- F2 screenshots freeze your game
+- AMD gpus
+- Some AMD cpus
+- Mip levels
 
 ## Community
 We have a [Discord server](https://discord.gg/H93wJePuWf) where you can track development progress, ask questions, or just hang out in.
 
+## Mod Dependencies
+[Fabric Language Kotlin](https://modrinth.com/mod/fabric-language-kotlin)
+
 ## Building
-1. Clone the repository (https://github.com/Blaze4D-MC/Blaze4D.git)
+1. Clone the repository (https://github.com/Blaze4D-MC/Blaze4D.git). Make sure to use `--recursive` to also clone [Rosella]{https://github.com/Blaze4D-MC/Rosella}.
```

Comment:
This addition is no longer needed because we use a maven repository

Reply:
Okay, fixed in the latest commit